kornia/filters/canny.py

```diff
@@ -52,8 +52,8 @@ def canny(
         >>> edges.shape
         torch.Size([5, 1, 4, 4])
     """
-    if not isinstance(input, torch.Tensor):
-        raise TypeError(f"Input type is not a torch.Tensor. Got {type(input)}")
+    if not isinstance(input, Tensor):
+        raise TypeError(f"Input type is not a Tensor. Got {type(input)}")
```

Comment:
update with CHECK_API ?

Reply:
True, I will try to apply check api along the filters module